tests/ltp/patches/umount2_01.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/testcases/kernel/syscalls/umount2/umount2_01.c b/testcases/kernel/syscalls/umount2/umount2_01.c
+index 46a6d59c3..e1057c2f9 100644
+--- a/testcases/kernel/syscalls/umount2/umount2_01.c
++++ b/testcases/kernel/syscalls/umount2/umount2_01.c
+@@ -22,6 +22,10 @@
+  *   point ceases to be busy."
+  */
+ 
++/* Patch to use root file system for the test as loop device file 
++ * system cannot be used as lkl Kernel Memory is set to 32M.
++ */
++
+ #include <errno.h>
+ #include <sys/mount.h>
+ 
+@@ -43,8 +47,8 @@ int TST_TOTAL = 1;
+ static int fd;
+ static int mount_flag;
+ 
+-static const char *device;
+-static const char *fs_type;
++static const char *device = "/dev/vda";
++static const char *fs_type = "ext4";
+ 
+ int main(int ac, char **av)
+ {
+@@ -71,7 +75,7 @@ static void setup(void)
+ 	tst_sig(NOFORK, DEF_HANDLER, NULL);
+ 
+ 	tst_tmpdir();
+-
++/*
```

Comment:
Please remove the not needed code instead of commenting out. Cod eget messy with not needed and commented out code.

Reply:
will update